pyproject.toml

```diff
@@ -0,0 +1,119 @@
+[build-system]
+requires = ["setuptools >= 61.2.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "xcube-core"
```

Comment:
In `pyproject.toml`, the name field cannot be dynamic (like e.g. "version"). 
I set the name to "xcube-core" since this will be name shown on [PyPI](https://pypi.org/project/xcube-core/). 
The import runs as usual with `import xcube` 

Will this name field in `pyproject.toml` have any implication with conda releases?